test/tests.js

```diff
@@ -174,6 +174,33 @@ QUnit.test('API for changing defaults', function (assert) {
 	assert.ok(Cookies.set('c', 'v').match(/path=\//), 'should roll back to the default path');
 });
 
+QUnit.test('falsy secure value', function (assert) {
+	var falsyValues = [
+		false,
+		null,
+		undefined,
+		0,
+		NaN,
+		''
+	];
```

Comment:
It is not necessary to test for all falsy values, just for the primitive `false`. As stated earlier only the primitive `Boolean` is supported.